js/ui/overlay/ui.overlay.js

```diff
@@ -155,6 +156,10 @@ const Overlay = Widget.inherit({
             // NOTE: private options
             hideTopOverlayHandler: () => { this.hide(); },
             hideOnParentScroll: false,
+
+            preventScrollEvents: true,
```

Comment:
Don't forget to disable it for a scheduler appointment form

Reply:
I have disabled it already.